src/content/docs/new-relic-solutions/solve-common-issues/diagnostics-cli-nrdiag/interpret-nrdiag-output.mdx

```diff
@@ -155,7 +155,7 @@ The Diagnostics CLI returns the following status codes after running:
 
 You can examine the `nrdiag-output.json` output in the UI. The account you select is associated with the license key that was validated by the Diagnostics CLI for upload.
 
-1. Select an app in [in the UI](https://one.newrelic.com), then search for **Diagnostics CLI Output**. OR, click this [**Diagnostics CLI Output** permalink](https://onenr.io/0x0jldL3EjW) to go directly to the page.
+1. Select an app [in the UI](https://one.newrelic.com), then search for **Diagnostics CLI Output**. OR, click this [**Diagnostics CLI Output** permalink](https://onenr.io/0x0jldL3EjW) to go directly to the page.
```

Comment:
```suggestion
1. Select an app [in the UI](https://one.newrelic.com), then search for **Diagnostics CLI Output**.
```
I feel like we should offer only one method to get to a page, even if there are other ways to get there. This method allows them to first pick their relevant app, vs. the option which is unclear which app it opens with.